src/pages/deployer/index.tsx

```diff
@@ -158,43 +213,69 @@ function Description() {
         <Link target="_blank" href="https://ton.org">
           TON blockchain
         </Link>
-        . This tool uses the{" "}
-        <Link href="https://github.com/ton-blockchain/TEPs/blob/master/text/0089-jetton-wallet-discovery.md">
-          discoverable jetton wallet
-        </Link>{" "}
-        protocol. This free educational tool allows you to deploy your own Jetton to mainnet in one
-        click. You will need at least 0.25 TON for deployment fees. <br />
+        . This free educational tool allows you to deploy your own Jetton to mainnet in one click.
+        You will need at least 0.25 TON for deployment fees. <br />
         <Spacer />
         For detailed instructions and in-depth explanations of all fields please see the{" "}
-        <Link
-          target="_blank"
-          href="https://github.com/ton-defi-org/jetton-deployer-contracts#jetton-metadata-field-best-practices">
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter-contract">
           GitHub README
         </Link>
         . It includes several best practice recommendations so please take a look.
         <Spacer />
         Never deploy code that you've never seen before! This deployer is fully open source with all
         smart contract code{" "}
-        <Link target="_blank" href="https://github.com/ton-defi-org/jetton-deployer-contracts">
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter-contract">
           available here
         </Link>
         . The HTML form is also{" "}
-        <Link target="_blank" href="https://github.com/ton-defi-org/jetton-deployer-webclient">
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter">
           open source
         </Link>{" "}
         and served from{" "}
-        <Link target="_blank" href="https://ton-defi-org.github.io/jetton-deployer-webclient">
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter">
           GitHub Pages
         </Link>
         . <Spacer />
-        Is this deployer safe? Yes! Read{" "}
-        <Link
-          target="_blank"
-          href="https://github.com/ton-defi-org/jetton-deployer-contracts#protect-yourself-and-your-users">
-          this
+        Is this deployer safe? Yes!{" "}
+        <Link target="_blank" href="https://github.com/ton-blockchain/minter-contract">
+          Read
         </Link>{" "}
-        to understand why.
+        this to understand.
```

Comment:
restore "why"

Reply:
Are you sure? Text was modified according to the Figma template

Reply:
For the moment, yes - figma usually shouldn't reflect product/copy decisions. I might be wrong on this specific one but I'll verify this alongside some other new texts

Reply:
restored